tests/entrypoints/openai/test_openai_schema.py

```diff
@@ -44,6 +44,6 @@ def get_schema(server):
 
 @schema.parametrize()
 @schema.override(headers={"Content-Type": "application/json"})
-async def test_openapi_stateless(case):
+def test_openapi_stateless(case):
```

Comment:
The type hint will help us see in IDEs that this function is not async

```suggestion
def test_openapi_stateless(case: schemathesis.Case):
```